docker/grade.sh.template

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+################################################################################
+## This shell script wraps the sbt invocation to set the arguments up         ##
+## properly, and redirects the noisy stdout to stderr, and stderr to stdout   ##
+## so that Coursera's grading systems can pick up the grades and feedback.    ##
+##                                                                            ##
+## Important security assumption: nothing except the grade output must be     ##
+## allowed to go to stderr, especially nothing controlled by the submission.  ##
+##                                                                            ##
+## Initial Author: Brennan Saeta <saeta@coursera.org>                         ##
+## For background / further details, please see:                              ##
+##    https://github.com/sbt-coursera/sbt-coursera                            ##
+################################################################################
+
+# Initialize the part id with a default value
+PART_ID="02Vi5q7m"
```

Comment:
Having a default value might not be a good idea. If we do not get the id we want this script to fail instead of running a randomly predefined task. Am I missing something?